sonar-plugin/sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/eslint/EslintBridgeServerImpl.java

```
@@ -454,6 +478,14 @@ private URI url(String endpoint) {
     }
   }
 
+  boolean nodePortIsProvided() {
+    return System.getenv(SONARJS_EXISTING_NODE_PROCESS_PORT) != null;
+  }
+
+  int nodeAlreadyRunningPort() throws NumberFormatException {
+    return Integer.parseInt(System.getenv(SONARJS_EXISTING_NODE_PROCESS_PORT));
```

catch the `NumberFormatException` and re-throw as  `IllegalStateException` , you can also do the range checks here and just throw `IllegalStateException` if value is invalid